src/2d/Sprite.cpp

```diff
@@ -1,5 +1,32 @@
 #include "Sprite.hpp"
+#include "gfx/Vertices.hpp"
+#include "sokol_gfx.h"
+#include "en/ComponentManager.hpp"
+
 namespace ari::en
 {
+	ARI_COMPONENT_IMP(Sprite)
+
+	static gfx::PosVertex2D s_SpritePosVertices[] =
+	{
+		{-1.0f, -1.0f},
+		{1.0f, -1.0f},
+		{1.0f, 1.0f},
+		{-1.0f, 1.0f},
+	};
```

Comment:
Where is the texcoords?
start with 0 instead of -1

Reply:
I've not written it yet! 
So the first vertex should be on (0,0) and (1,0), (1,1), (0,1)?

Reply:
yes and don't forget the texcoords